content/en/docs/contribute/intermediate.md

```diff
@@ -911,8 +911,7 @@ deadlines. Some deadlines related to documentation are:
 
 If your feature is an Alpha feature and is behind a feature gate, make sure you
 add it to [Feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
-as part of your pull request. If your feature is moving out of Alpha, make sure to
-remove it from that file.
+as part of your pull request. If your feature is moving to Beta or to General Availability, update the feature gates file.
```

Comment:
@divya-mohan0209 I would preserve the wrapping:
```suggestion
as part of your pull request. If your feature is moving to Beta or to
General Availability, update the feature gates file.
```

Reply:
Alright, sure.

Reply:
Please review now